adflow/pyADflow.py

```diff
@@ -3658,7 +3677,7 @@ def _setAeroProblemData(self, aeroProblem, firstCall=False):
             self.adflow.bcdata.setbcdata(nameArray, dataArray, groupArray, 1)
 
         if not firstCall:
-            self.adflow.initializeflow.updatebcdataalllevels()
+            self.adflow.initializeflow.updatebcdataalllevels(printInfo)
```

Comment:
Also here, instead of passing boolean values here (and elsewhere in Fortran), turning off BC warning should be done and controlled by an option, which sets a module variable in the Fortran layer. 
Then its just an `use` in the subroutines that need access to this flag.
There already exists a [similar variable](https://github.com/mdolab/adflow/blob/e7ada3de62b6500b574545b4f1c7f04c01676bf9/src/modules/inputParam.F90#L286C16-L286C29) `printWarnings`  which is similar in function, but for negative volumes. I think a `printBCWarnings` variable (or similar) would make more sense here.